desktop-environment.el

```diff
@@ -361,6 +389,27 @@ the screen."
     (async-shell-command desktop-environment-screenlock-command)))
 
 
+
+;;; Commands - keyboard layout
+
+;;;###autoload
+(defun desktop-environment-keyboard-layout-cycle-forward ()
+  "Cycle in desktop-environment-keyboard-layouts-ring by one forward."
```

Comment:
I would mention `desktop-environment-keyboard-layouts` instead because the ring is private. Please also add quoting around the variable name.